traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -135,28 +135,29 @@ RETURNING
 	start_time as startTime
 `
 
-const revalQuery = `
-UPDATE server SET %s=TRUE
-WHERE server.status NOT IN (
-                             SELECT status.id
-                             FROM status
-                             WHERE name IN ('OFFLINE', 'PRE_PROD')
-                           )
+const queueUpdateOrRevalQuery = `
+UPDATE public.server
+SET %s = now()
+WHERE server.status IN (
+						SELECT status.id
+						FROM status
+						WHERE name IN ('ONLINE', 'REPORTED', 'ADMIN_DOWN')
+						)
      AND server.profile IN (
-                             SELECT profile_parameter.profile
-                             FROM profile_parameter
-                             WHERE profile_parameter.parameter IN (
-                                                                    SELECT parameter.id
-                                                                    FROM parameter
-                                                                    WHERE parameter.name='location'
-                                                                     AND parameter.config_file='regex_revalidate.config'
-                                                                  )
+							SELECT profile_parameter.profile
+							FROM profile_parameter
+							WHERE profile_parameter.parameter IN (
```

Comment:
the indentation of this query seems to have gotten a little out of hand 😄

Reply:
ha! Believe it or not I actually reduced it some. I'll reign it in a bit more.